cmd/convox/start.go

```diff
@@ -84,7 +84,9 @@ func cmdStart(c *cli.Context) error {
 	}
 
 	cache := !c.Bool("no-cache")
-	r := m.Run(dir, app, cache)
+	sync := c.Bool("sync")
```

Comment:
yeah i thought that was odd, but i went with what was there originally.
Will change it over for consistency